BookingQueueSubscriber/BookingQueueSubscriber.Services/MessageHandlers/HearingParticipantsUpdatedHandler.cs

```diff
@@ -33,6 +37,12 @@ public async Task HandleAsync(HearingParticipantsUpdatedIntegrationEvent eventMe
             };
 
             await _videoApiService.UpdateConferenceParticipantsAsync(conferenceResponse.Id, updateConferenceParticipantsRequest);
+
+            if(updateConferenceParticipantsRequest.NewParticipants.Count > 0 && conferenceResponse.ScheduledDateTime.Date == DateTime.Today)
```

Comment:
Limited to today as VIH-7558 needs to also update VHOs who will see all hearings for the current date.